src/test/resources/mocks/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM rodolpheche/wiremock
```

Comment:
What is this for ? Is this something extra we need to run / build a docker image for ? Does it need mention in the README ?

Reply:
I added this for some debugging I was doing just to make it easier to run a standalone Wiremock instance for the integration tests - we don't really need it though so I'll remove it